frontend/cypress/integration/login_page_spec.js

```diff
@@ -4,9 +4,9 @@ describe("The Login Page", function() {
     cy.get('input[name="email"]').type("Nonexistent email");
     cy.get('input[name="password"]').type("password");
     cy.get('button[type="submit"]').click();
-
-    cy.wait(3000);
-
-    cy.url().should("include", "/login");
+    cy.get(".MuiFormHelperText-required-211").should(
```

Comment:
šitas selectorius nepraeina ant production build, pakeisk į kažką univeraslesnio
For reference iš moneybunney.tk, šitas elementas atrodo maždaug taip:
`<p class="jss204 jss205 jss210 jss211">Wrong password</p>`

Reply:
Dar noriu pridurt, kad pas mane ir ant lokalaus development build nepraeitų šitas testas, nes tas elementas atrodo taip:
`<p class="MuiFormHelperText-root-260 MuiFormHelperText-error-261 MuiFormHelperText-filled-266 MuiFormHelperText-required-267">Wrong password</p>`
In summary, geriau nedaryk selectorių pagal klases, kurių mes nerašėm rankom 😃 
_Kadangi MUI ateina su JSS, tai visos klasės kompiliuojamos ir keičia savo pavadinimus pastoviai, daugiau apie tai gali rast čia: https://cssinjs.org/features_

Reply:
Nelabai gerai dependinti ant tokios implementacijos detales

Reply:
Taaip, visiškai pritariu, bet neturėjau visai idėjų. Dabar padariau paprastą, kad expectinu, jog visame page atsirastų "Wrong password". Pacommitinau, pachekinkit.

Reply:
nėra siaubingas approach, gražiai skaitosi 👍

Reply:
Uhm, o tai jis veikia ant kitu kompu? :D Pavyzdziui ant production build?